src/router/authGuard.js

```diff
@@ -0,0 +1,25 @@
+import { useAuthStore } from '@/store/auth.js';
+import authRoutes from './auth';
+import userRoutes from './user';
+
+const getRouteNames = routes => {
+  const routeNames = routes.reduce((routeNames, route) => {
+    if (route.name) routeNames.push(route.name);
+    return routeNames;
+  }, []);
+  return routeNames;
```

Comment:
Might as well return it immediately
```suggestion
  return routes.reduce((routeNames, route) => {
    if (route.name) routeNames.push(route.name);
    return routeNames;
  }, []);
```

Reply:
I'm always in that little moments of returning the only value being processed in a function not so sure which way would be the better one - "const-ing" it or not 😅

I'm ok with either of two so I'll go with your suggestion 👍